### ISSUE TITLE
[Feature Request] Batch WAV Export

### ISSUE THREAD

**Issue by jadasse** [BleuBleu/FamiStudio#187]
Can FamiStudio have the feature to export all songs within a document instead of one at a time? They can all have the same exporting settings

**Comment by BleuBleu** [comment]
What export format are we talking about. Please when you open issues, be specific, otherwise ill just ignore them.

**Comment by jadasse** [comment]
Would it matter what format to export? If so, WAV files because they're more natural, in my opinion.

**Comment by BleuBleu** [comment]
Yeah it does, because some formats (NSF, ROM, FamiStudio text, etc.) supports it. It is currently not supported for WAV/MP3, you could use the command line to do a batch export as a workaround.

-Mat

**Comment by jadasse** [comment]
> Yeah it does, because some formats (NSF, ROM, FamiStudio text, etc.) supports it. It is currently not supported for WAV/MP3, you could use the command line to do a batch export as a workaround.
> 
> -Mat

OK

**Comment by jadasse** [comment]
Are there specific settings for that or is it the same "for %i in" type?

**Comment by BleuBleu** [comment]
Could you try something like this? Of course a loop would be nicer.

```
FamiStudio.exe "Demo Songs\Nice.fms" wav-export song0.wav -export-song:0 -wav-export-rate:48000
FamiStudio.exe "Demo Songs\Nice.fms" wav-export song1.wav -export-song:1 -wav-export-rate:48000
FamiStudio.exe "Demo Songs\Nice.fms" wav-export song2.wav -export-song:2 -wav-export-rate:48000
FamiStudio.exe "Demo Songs\Nice.fms" wav-export song3.wav -export-song:3 -wav-export-rate:48000
FamiStudio.exe "Demo Songs\Nice.fms" wav-export song4.wav -export-song:4 -wav-export-rate:48000
```

**Comment by BleuBleu** [comment]
Ill keep it open since this is a legitimate request. :)

**Comment by jadasse** [comment]
> Ill keep it open since this is a legitimate request. :)

OK
